src/containers/milestone.py

```diff
@@ -7,7 +7,7 @@ def __init__(self):
         self.title: Optional[str] = None
         self.html_url: Optional[str] = None
 
-    def load_from_json(self, milestone):
+    def load_from_api_json(self, milestone):
```

Comment:
Why we call is load_from_api_json and not load_from_json?

Reply:
I already responed to the project pattern in:
https://github.com/AbsaOSS/living-doc-generator/pull/16#discussion_r1658276252

Reply:
https://github.com/AbsaOSS/living-doc-generator/pull/16#discussion_r1680645735